src/native/libs/System.Globalization.Native/static-build/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.6)
+
+project(System.Globalization.Native.User.Machine.Build C)
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux") # Only Linux for now
+
+configure_file(../config.h.in config.h)
+
+set(NATIVEGLOBALIZATION_SOURCES
+  ../pal_calendarData.c
+  ../pal_casing.c
+  ../pal_collation.c
+  ../pal_idna.c
+  ../pal_locale.c
+  ../pal_localeNumberData.c
+  ../pal_localeStringData.c
+  ../pal_normalization.c
+  ../pal_icushim_static.c
+  ../pal_timeZoneInfo.c
+)
+
+add_library(System.Globalization.Native
+    STATIC
+    ${NATIVEGLOBALIZATION_SOURCES}
+)
+
+add_compile_options(-Wno-switch-enum)
+add_compile_options(-Wno-covered-switch-default)
+
+# Workaround for warnings produced by ICU headers
+add_compile_options(-Wno-reserved-id-macro)
+add_compile_options(-Wno-documentation)
+add_compile_options(-Wno-documentation-unknown-command)
+
+# Workaround for https://unicode-org.atlassian.net/browse/ICU-20601
+add_compile_options(-Wno-extra-semi-stmt)
+add_compile_options(-Wno-unknown-warning-option)
+
+add_definitions(-D_GNU_SOURCE -g -fPIC -DHOST_64BIT -DHOST_AMD64 -DHOST_UNIX -DPALEXPORT=EXTERN_C -DTARGET_64BIT -DTARGET_AMD64 -DTARGET_LINUX -DTARGET_UNIX -D_FILE_OFFSET_BITS=64 -DSTATIC_ICU -DSTATIC_SHIM_COMPILE)
```

Comment:
`-g` and `-fPIC` are not definitions and listed in `target_compile_options`. Do we need them here?